File: eform-client/src/app/plugins/modules/workorders-pn/models/work-order.model.ts
import {CommonImageModel} from 'src/app/common/models';

export class WorkOrderModel {
  id: number;
  createdAt: Date | string;
  createdBy: string;
  description: string;
  correctedAtLatest: Date;
  doneAt: Date | string;
  doneBy: string;
  picturesOfTask: CommonImageModel[] = [];
  picturesOfTaskDone: CommonImageModel[] = [];
  descriptionOfTaskDone: string;
}

File: eFormAPI/Plugins/WorkOrders.Pn/WorkOrders.Pn/Infrastructure/Models/WorkOrderModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WorkOrders.Pn.Infrastructure.Models
{
    public class WorkOrderModel
    {
        public int Id { get; set; }
        public DateTime CreatedAt { get; set; }
        public int CreatedByUserId { get; set; }
        public string Description { get; set; }
        public DateTime CorrectedAtLatest { get; set; }
        public DateTime DoneAt { get; set; }
        public int DoneBySiteId { get; set; }
        public string DescriptionOfTaskDone { get; set; }
    }
}
